core/src/test/java/org/lflang/tests/compiler/LinguaFrancaValidationTest.java

```diff
@@ -78,6 +79,14 @@
 @InjectWith(LFInjectorProvider.class)
 public class LinguaFrancaValidationTest {
 
+  /**
+   * This is a list of tests that have no representation in LF syntax, possibly because we are
+   * moving away from keeping these in LF files and instead are moving toward keeping them in
+   * configuration files.
+   */
+  private static final List<TargetProperty<?, ?>> propertiesWithoutLfSyntaxRepresentation =
```

Comment:
This looks like unnecessary bookkeeping to me. If the `fromAst` method throws an exception, then that exception can just be caught and dealt with, no?

Reply:
But the test framework needs to know whether that exception indicates a test failure, or if it is the intended behavior.

Reply:
In the end I realized that I could not use an exception here because if this test sees the exception, then that means the validator crashes instead of working normally and producing an error message on the correct line.

However, the change that I did make was in the spirit of the suggestion. I report an error message on the correct line, and in the test I check that the error message is there.